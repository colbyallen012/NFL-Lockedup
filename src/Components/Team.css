.team {
  background-color: lightcyan;
  color: black;
  margin: 10px;
  width: 200px;
  height: 200px;
}
