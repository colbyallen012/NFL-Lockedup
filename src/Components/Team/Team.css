.team {
  background-color: white;
  border-radius: 20px;
  color: black;
  margin: 10px;
  padding: 5px;
  width: 90%;
}

.player {
  margin: 0px auto;
  text-align: center;
  width: 50%;
}

.bolder {
  font-weight: bolder;
}

.lighter {
  font-weight: normal;
}
