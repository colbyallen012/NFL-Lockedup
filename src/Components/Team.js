import React from 'react';
import { Route, Link } from 'react-router-dom'
import './Team.css';

const Team = ({city, name, conference, abrv}) => {
  return (
    <Route exact path='/' render={() =>
    <div className='team'>
      <h2>{city}</h2>
      <p>{name}</p>
      <p>{conference}</p>
      <Link to={`/${abrv}/Arrests`}>
        <button>More Info</button>
      </Link>
    </div>
    }/>
  )
}

export default Team;