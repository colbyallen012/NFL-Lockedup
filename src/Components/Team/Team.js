import React from 'react';
import './Team.css';

const Team = ({player, position, arrest, description, outcome, season}) => {

  return (
    <div className='team'>
      <h3 className='player-name'>{player} <span className='lighter'>{position}</span></h3>
      <p className='info'><span className='bolder'>Charge: </span>{arrest}</p>
      <p className='info'><span className='bolder'>Outcome: </span>{outcome}</p>
      <p className='info'><span className='bolder'>Description: </span>{description}</p>
      <p className='info'><span className='bolder'>Season: </span>{season}</p>
    </div>
  )
}

export default Team;