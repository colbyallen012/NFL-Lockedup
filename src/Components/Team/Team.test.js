import React from 'react';
import { shallow } from 'enzyme';
import Team from './Team';

describe('Team', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <Team 
        player='Colby Allen' 
        position='QB'
        arrest='Bank Robbery'
        description='Robbed a bank for 100 million dollars'
        outcome='Still on the lamb'
        season='Off season' 
        key={1}
      />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})