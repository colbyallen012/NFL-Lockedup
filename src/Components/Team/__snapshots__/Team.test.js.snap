// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Team should match the snapshot 1`] = `
<div
  className="team"
>
  <h3>
    Colby Allen
  </h3>
  <p>
    <span
      className="bolder"
    >
      QB
    </span>
  </p>
  <p>
    <span
      className="bolder"
    >
      Charge: 
    </span>
    Bank Robbery
     
    <span
      className="bolder"
    >
      Outcome: 
    </span>
    Still on the lamb
  </p>
  <p>
    <span
      className="bolder"
    >
      Description: 
    </span>
    Robbed a bank for 100 million dollars
  </p>
  <p>
    <span
      className="bolder"
    >
      Season: 
    </span>
    Off season
  </p>
</div>
`;
